src/main/java/CodeBuildException.java

```diff
@@ -0,0 +1,14 @@
+import org.jenkinsci.plugins.scriptsecurity.sandbox.whitelists.Whitelisted;
+
+public class CodeBuildException extends InterruptedException{
```

Comment:
This extends `InterruptedException` because `AbortException` is final/closed for inheritance. Jenkins seems to handle `InterruptedException` the same way as `AbortException`